tools/benchmark/proto/buf.yaml

```diff
@@ -0,0 +1,19 @@
+version: v1
+name: buf.build/mods/consensus
```

Comment:
can we call this benchmarks?
I don't think we need to publish those protos on buf right?

Reply:
oops, copy pasted error.  
I don't think we need to publish them. what should I change to prevent that?

Reply:
Nothing, I won't create a module on buf.build then 👍

Maybe we can add a comment in there to explain that too.